src/Language/PureScript/Bridge/SumType.hs

```diff
@@ -109,17 +144,19 @@ data Instance (lang :: Language)
   | Enum
   | Bounded
   | HasConstrIndex
+  | ToData
```

Comment:
Do we need this? I.e. do we really expect anyone using the library to want to generate HasConstrIndices instances but not To/FromData instances? 

Either way I think `mkSumTypeIndexed` should generate `ToData/FromData` instances by default. Maybe we should rename it to `mkPlutusSum` or something like that?

Reply:
I completely agree!